index.html

```diff
@@ -208,6 +208,19 @@ <h4 class="social-links">LET’S CONNECT</h4>
       </ul>
     </div>
   </section>
+  <section class="contact">
+    <div class="form-title">
+      <h1>Contact me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form class="form" action="https://formspree.io/f/meqwkove" method="post"></form>
+      <input type="text" name="first_name" maxlength="30" required placeholder="  Your Name">
+      <input type="email" name="email" id="email" required placeholder="  Your Email Address">
+      <textarea name="message" id="text" maxlength="500" required placeholder="Write your message here"></textarea>
+      <button type="submit">Get in touch</button>
```

Comment:
Please consider wrapping all the form elements in the form tag. Currently, the form element is not holding any other elements which are making the form submission not possible.

## Consequences:-

As your form is not submitting so none of the validations are working obviously and that's why I am not able to verify them as a Code Reviewer. So consider them also when you do the changes to your form to make all of them work.

Reply:
Hi **@ABDULALI3468** 

Thanks for your time and for reviewing my code. I have fixed the form issue. 🙏